# Define the goal
float32 goal_angle  # Specify which dishwasher we want to use
---
# Define the result
uint32 goal_pose_id
---
# Define a feedback message 
float32 current_angle
